src/tokens.ts

```diff
@@ -85,7 +85,7 @@ export class TokenTransfer {
         }
     }
 
-    static newFromEgldAmount(amount: bigint): TokenTransfer {
```

Comment:
maybe this should have been deprecated as well? So it is not breaking.

Reply:
we discussed in the office and said it is ok to rename it as it was not long time added